### PR TITLE
Port to QBI first pass

### PR DESCRIPTION
**Pull request by jamiebrynes7**
#### Description
This PR converts the Playground project to use QBI. It does this in the most naive way possible by emulating the old entity interest system. I.e. - checkout any component in a 25 meter radius. 

#### Tests

- [x] Local deployment w/ two workers.
- [x] Cloud deploy

#### Documentation

- [x] Changelog

### CODE REVIEW

File: workers/unity/spatialos.MobileClient.worker.json [hunk 1]
```diff
@@ -21,14 +21,9 @@
...
     },
-    "entity_interest": {
-      "range_entity_interest": {
-        "radius": 25
-      }
-    },
     "component_delivery": {
       "default": "RELIABLE_ORDERED",
-      "checkoutAllInitially": true
+      "checkoutAllInitially": false
```
Comment (zeroZshadow) [inline workers/unity/spatialos.MobileClient.worker.json]: Is the `component_delivery` part required? Or can it be removed entirely?

Comment (jamiebrynes7) [reply workers/unity/spatialos.MobileClient.worker.json]: The docs say to keep it https://docs.improbable.io/reference/14.4/shared/authority-and-interest/interest/chunk-based-interest-cbi#turning-off-cbi

File: workers/unity/Assets/Playground/Editor/Improbable.Playground.Editor.asmdef [hunk 2]
```diff
@@ -4,9 +4,10 @@
         "Improbable.Playground",
         "Improbable.Gdk.Core",
         "Improbable.Gdk.Generated",
-        "Improbable.Gdk.TransformSynchronization"
+        "Improbable.Gdk.TransformSynchronization",
+        "Improbable.Gdk.QueryBasedInterestHelper",
+        "Unity.Entities"
```
[the remark this answers is not in the document]
Comment (jamiebrynes7) [reply workers/unity/Assets/Playground/Editor/Improbable.Playground.Editor.asmdef]: There was a reason this was added, but I can't recall. Let me double check

[the remark this answers is not in the document]
Comment (jamiebrynes7) [reply workers/unity/Assets/Playground/Editor/Improbable.Playground.Editor.asmdef]: Its required for the compiler to resolve the overloads. Pushed a repro case to https://github.com/spatialos/gdk-for-unity/tree/bug/interest-query-compiler-nonsense

**Pull request merged**